src/Commands/IndexProductsCommand.php

```diff
@@ -66,30 +56,31 @@ public function handle()
 
                 $showOutOfStock = (bool) Rapidez::config('cataloginventory/options/show_out_of_stock', 0);
 
-                $productQuery->chunk($this->chunkSize, function ($products) use ($store, $bar, $index, $categories, $showOutOfStock) {
-                    foreach ($products as $product) {
+                $productQuery->chunk($this->chunkSize, function ($products) use ($store, $bar, $categories, $showOutOfStock) {
+                    $this->indexer->index($products, function ($product) use ($store, $categories, $showOutOfStock) {
                         if (!$showOutOfStock && !$product->in_stock) {
-                            continue;
+                            return null;
```

Comment:
```suggestion
                            return;
```

Does the same right?